.github/workflows/test-postgresql.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: PDB tests with PostgreSQL
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: Cache Maven packages
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+
+    - name: Test with PostgreSQL
+      run: mvn -B clean test -Ppostgresql
```

Comment:
Question: On average, how much time will tests take in each job?

Reply:
A few minutes. PostgreSQL is 3 minutes. But DB2 locally takes 10 minutes.

Reply:
I was thinking in running tests for each profile but 10 minutes is probably too much

Reply:
We will :) but each profile can go into a single workflow, this way they can run in parallel